pkgs/top-level/all-packages.nix

```diff
@@ -9814,6 +9821,8 @@ in
   gcc9Stdenv = overrideCC gccStdenv buildPackages.gcc9;
   gcc10Stdenv = overrideCC gccStdenv buildPackages.gcc10;
 
+  fastStdenv = overrideCC gccStdenv (wrapNonDeterministicGcc gccStdenv buildPackages.gcc10);
```

Comment:
I don't think we should provide this attribute since 1) it's undiscoverable; 2) there is a risk that people start using it in nixpkgs (or elsewhere), and then we end up with closure bloat and libgcc/libstdc++ conflicts; 3) building gcc is expensive.

Reply:
I made it more discoverable and documented it [here](https://github.com/NixOS/nixpkgs/pull/120775) and [here](https://nixos.wiki/wiki/C#Faster_GCC_compiler)